temporalcli/commandsmd/commands.md

```diff
@@ -427,6 +430,8 @@ temporal workflow start \
 * `--search-attribute` (string[]) - Passes Search Attribute in key=value format. Use valid JSON formats for value.
 * `--memo` (string[]) - Passes Memo in key=value format. Use valid JSON formats for value.
 * `--fail-existing` (bool) - Fail if the workflow already exists.
+* `--start-delay` (duration) - Specify a delay before the workflow starts. Cannot be used with a cron schedule. If the
+  workflow receives a signal or update before the delay has elapsed, it will begin immediately.
```

Comment:
Can I confirm -- queries don't have this effect?

Reply:
I don't believe so, but I just took this content verbatim from #402. I'd have to defer to start delay implementers to be sure, but https://github.com/temporalio/api/pull/264 implies just signal (they probably forgot to add update).